pennylane/ftqc/parametric_midmeasure.py

```diff
@@ -146,9 +395,141 @@ def label(self, decimals=None, base_label=None, cache=None):  # pylint: disable=
         return _label
 
 
+class XMidMeasureMP(ParametricMidMeasureMP):
+    """A subclass of ParametricMidMeasureMP that uses the X measurement basis
+    (angle=0, plane="XY"). For labels and visualizations, this will be represented
+    as a X measurement. It is otherwise identical to the parent class."""
+
+    _shortname = "measure_x"
+
+    # pylint: disable=too-many-arguments
+    def __init__(
+        self,
+        wires: Optional[Wires],
+        reset: Optional[bool] = False,
+        postselect: Optional[int] = None,
+        id: Optional[str] = None,
+    ):
+        super().__init__(
+            wires=Wires(wires), angle=0, plane="XY", reset=reset, postselect=postselect, id=id
+        )
+
+    def _flatten(self):
+        metadata = (
+            ("wires", self.raw_wires),
+            ("reset", self.reset),
+            ("id", self.id),
+        )
+        return (None, None), metadata
+
+    def __repr__(self):
+        """Representation of this class."""
+        return f"{self._shortname}(wires={self.wires.tolist()})"
+
+    def label(
+        self, decimals: float = None, base_label: str = None, cache: dict = None
+    ):  # pylint: disable=unused-argument
+        r"""How the mid-circuit measurement is represented in diagrams and drawings.
+
+        Args:
+            decimals=None (Int): If ``None``, no parameters are included. Else,
```

Comment:
I might be wrong, but I think Sphinx already captures the default parameter value in the function call signature. Similarly for the parameters below. I also changed `Int` -> `int`.

```suggestion
            decimals (int): If ``None``, no parameters are included. Else,
```

Reply:
Yep that is correct, no need to specify it here

Reply:
updated!